src/test/java/PlayAgainMenuTest.java

```diff
@@ -0,0 +1,47 @@
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class PlayAgainMenuTest {
+    @Test
+    public void itWillDisplayTheMenu() {
+        FakeUI ui = new FakeUI();
+        ui.input.add("Y");
+        ui.input.add("N");
+
+        new PlayAgainMenu(ui).display();
+
+        assertEquals("Play again?\nY/N:  Do you want to use a different game configuration?\nY/N:  ", ui.log);
+    }
+
+    @Test
+    public void itWillOnlyAcceptAcceptYOrN() {
+        FakeUI ui = new FakeUI();
+        ui.input.add("asdk");
+        ui.input.add("0091");
+        ui.input.add("N");
```

Comment:
Also, I have used a "log" for testing the components that interact with the screen a couple of times now.  Do either of you have any opinions on that approach?